### DISCUSSION TITLE
The AudioContext was not allowed to start

### DISCUSSION THREAD

**Discussion by claroja** [CoderLine/alphaTab#698] [Q&A] [answered]
the chrome reports:
```
The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page.
```
at `return new AudioContext()` in
```
class AlphaSynthWebAudioOutputBase {
    ......
        createAudioContext() {
            if ('AudioContext' in Environment.globalThis) {
                return new AudioContext();
            }
    ......
```
How can I fix it? Thanks.

# Environment
chrome : 97.0.4692.71
@coderline/alphatab : ^1.2.1

**Comment by Danielku15** [comment, marked as answer]
There is nothing to fix. This is a warning that Chrome always logs. `It must be resumed (or created) after a user gesture on the page` <- this is exactly what alphaTab is doing. It resumes the context as part of different operations of one at least is expected to be trusted by the browser. This is usually the call to `play` or `playPause` as part of a user interaction or upon a general click or interaction with the page. 

There is this open Chrome request to get rid of the warning on construction: https://bugs.chromium.org/p/chromium/issues/detail?id=943258

**Reply by claroja** [reply]
Get it, thank you very much!
